index.html

```diff
@@ -175,6 +175,178 @@ <h5 class="сustomer-кeviews">Отзывы</h5>
+                        <label class="additional-services-label" for="cheackbox_subjects">Крупногабаритные предметы</label>
+                    </div>
+                </div>
+                <div class="time-button-wrapper">
+                    <p class="time">05:37:25</p>
+                    <div class="button-btn-order">
+                        <button class="btn-order">Заказать со скидой</button>
+                        <div class="shadow-btn-order"></div>
+                    </div>
+                </div>
+                <button class="btn-mobile-order">Перезвонить мне</button>
+                <div class="chekbox-data-processing">
+                    <input class="checkbox" type="checkbox" id="data-processing">
+                    <label class="checkbox-cheack" for="data-processing">Даю согласие на обработку данных</label>
+                </div>
+        </div>
+        <div class="map">
+            <iframe src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d1225.3825097941658!2d23.77834254427729!3d52.10220796637626!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x47210e8ecc2403f1%3A0x5d6ffdcabb18ea86!2z0YPQuy4g0JzQvtGB0LrQvtCy0YHQutCw0Y8gMjgxLCDQkdGA0LXRgdGC!5e0!3m2!1sru!2sby!4v1620162823274!5m2!1sru!2sby"
+                width="100%" height="100%" style="border:0;" allowfullscreen="" loading="lazy" class="map-locl"></iframe>
+            <div class="box-in-the-card">
+                <h3>MOVEE</h3>
+                <ul class="map-box-list">
+                    <li class="map-box-item">Брест</li>
+                    <li class="map-box-item"><a href="tel:8 (039)-999-12-13">8 (039)-999-12-13</a></li>
+                    <li class="map-box-item"><a href="mailto:hello@movee.ru">hello@movee.ru</a></li>
+                </ul>
+                <div class="social-networks">
+                    <a href="https://www.youtube.com/ " target="_blank">
+                        <img src="img/youtube.png " alt=" ">
+                    </a>
+                    <a href="https://vk.com/ " target="_blank ">
+                        <img src="img/vk.png " alt=" ">
+                    </a>
+                    <a href="https://facebook.com/ " target="_blank">
+                        <img src="img/facebook.png " alt=" ">
+                    </a>
+                    <a href="https://instagram.com/ " target="_blank">
+                        <img src="img/inst.png " alt=" ">
+                    </a>
+                </div>
+            </div>
+        </div>
+    </div>
+    <footer>
+        <div class="footer-wrapper-content">
+            <div class="footer-ifo-block">
+                <h3>Movee</h3>
+                <p class="footer-mobile-privacy">Политика конфиденциальнности</p>
+                <p class="footer-mobile-privacy-agree">Соглашение на обработку персональных данных</p>
+            </div>
+            <div class="footer-nav-list">
```

Comment:
This is an redundant wrapper. Why is this here? What's the logic in using this div?